BedrockPlugin.cpp

```diff
@@ -2,7 +2,7 @@
 #include "BedrockPlugin.h"
 
 // Global static values
-list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = 0;
+list<BedrockPlugin*>* BedrockPlugin::g_registeredPluginList = nullptr;
```

Comment:
Personally, I don't really like this syntax.  Maybe I'm old school, but I don't find `nullptr` to be more readable or informative than `0`.  Thoughts @tylerkaraszewski?